### ISSUE TITLE
Create a button to access the admin dashboard

### ISSUE THREAD

**Issue by Caleb-Cohen** [Together-100Devs/Together#405]
**Related Issues**
Project: Admin Dashboard
Depends on: #404 

**Description**
- Create a button on the homepage that will access the admin dashboard 
- Renders admin dashboard, which for this PR will be a blank page
- Button should be to the left of the home button

**Acceptance Criteria**
Button exists, is clickable, routes to blank page.

**Comment by trstngttschlk** [comment]
i'd like to work on this

**Comment by Caleb-Cohen** [comment]
> i'd like to work on this

Hey there! This depends on #404. #404 should be completed prior to this being assigned to anyone.

**Comment by Caleb-Cohen** [comment]
> i'd like to work on this

This is now open if you're interested!

**Comment by trstngttschlk** [comment]
i'd love to contribute and be assigned this issue

**Comment by Caleb-Cohen** [comment]
> i'd love to contribute and be assigned this issue

Assigned!

**Comment by emmebravo** [comment]
@DevinCLane @intelagense this issue's PR is awaiting review

**Comment by DevinCLane** [comment]
yeah all these project admin dashboard issues we will want to discuss to see what direction we move that in
